packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -57,7 +57,7 @@ const paginationWidthLarge = `calc(${paginationActiveBulletSize} + ${paginationB
   paginationVisibleBulletCount - 1
 } + ${paginationGapLarge} * ${paginationVisibleBulletCount - 1} + 2 * ${paginationInset})`; // Width for one active bullet + width of inactive bullets + spacing
 
-const selectorHeading = 'h2,::slotted([slot="heading"])';
+const selectorHeading = ':is(.heading)';
```

Comment:
Why do we need the `:is` selector here, doesn't selecting `.heading` would be enough?

Reply:
We need the `:is` selector to have the possibilities to combine selectors on `global` space, e.g. in line 121:
`...((hasHeading || hasDescription) && {`

Reply:
`:is(.heading),p,::slotted([slot="description"])` and `.heading,p,::slotted([slot="description"])` shouldn't make a difference